apps/coscrad-frontend/src/components/resources/bibliographic-references/bibliographic-reference-detail-thumbnail-presenters/court-case-bibliographic-reference-detail.thumbnail.presenter.tsx

```diff
@@ -1,29 +1,28 @@
 import {
     IBibliographicReferenceViewModel,
-    ICourtCaseBibliographicReferenceData
+    ICourtCaseBibliographicReferenceData,
+    ResourceType,
 } from '@coscrad/api-interfaces';
-import { Card, CardContent, CardHeader, Divider } from '@mui/material';
+import { Divider } from '@mui/material';
+import { ResourceDetailThumbnailPresenter } from 'apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views';
 import { Abstract } from './abstract';
 
 export const CourtCaseBibliographicReferenceDetailThumbnailPresenter = ({
     id,
     data: { caseName, abstract, dateDecided, pages },
 }: IBibliographicReferenceViewModel<ICourtCaseBibliographicReferenceData>): JSX.Element => {
+    const name = caseName;
+
     return (
-        <Card>
-            <CardHeader title="Court Case Bibliographic Reference"></CardHeader>
-            <CardContent>
-                <div data-testid={id}>
-                    {caseName}
-                    <br />
-                    {pages && <div>{pages} pages</div>}
-                    {dateDecided && <div>({dateDecided})</div>}
-                    <Divider />
-                    {abstract !== null && typeof abstract !== undefined && (
-                        <Abstract abstract={abstract} />
-                    )}
-                </div>
-            </CardContent>
-        </Card>
+        <ResourceDetailThumbnailPresenter
+            id={id}
+            name={name}
+            type={ResourceType.bibliographicReference}
+        >
+            {pages && <div>{pages} pages</div>}
```

Comment:
These are examples of cases where the `SinglePropertyPresenter` is not sufficient, or too generic.